### PR TITLE
snap-repair: add `snap-repair run`

### PR DESCRIPTION
**Pull request by mvo5**
This outlines the `snap-repair run` command as discussed in https://forum.snapcraft.io/t/repair-capability-emergency-fixes/311

Based on https://github.com/snapcore/snapd/pull/3211

### CODE REVIEW

File: cmd/snap-repair/repair.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+	if err := ioutil.WriteFile(r.ranStamp(), nil, 0644); err != nil {
+		logger.Noticef("cannot write stamp: %s", err)
+	}
+	logger.Noticef("finished repair %s, logs in ", r.ra.RepairID(), r.log())
+
+	return err
+}
+
+// FIXME: bypass the assertion DB entirely and collect all repair
+//        bits in /var/lib/snapd/repair/ ?
```
Comment (pedronis) [inline cmd/snap-repair/repair.go]: yes, something like that is probably what we want

**Comment by zyga** [comment]
You need to install the new tool:
```
dh_install: usr/bin/snap-repair exists in debian/tmp but is not installed to anywhere
```

**Comment by mvo5** [comment]
this branch has the wrong prefix, I close it and open with the right name in #3226. I will link to here.

File: cmd/snap-repair/repair.go (hunk 1 above)
Comment (mvo5) [inline cmd/snap-repair/repair.go]: Great, updated the FIXME.

**Comment by mvo5** [comment]
@zyga thanks! updated the code.

**Pull request closed without merging**